doc/lsst.dax.apdb_migrate/migrations/sql/schema.rst

```diff
@@ -8,9 +8,29 @@ Upgrade from 0.1.0 to 0.1.1
 Migration script: `schema_0.1.1.py <https://github.com/lsst-dm/dax_apdb_migrate/blob/main/migrations/sql/schema/schema_0.1.1.py>`_
 
 This migration creates ``metadata`` table to the schema which was added in version ``0.1.1`` of ``apdb.yaml``.
-It also populates metadata with version numbers for both ``schema`` and ``ApdbSql`` trees (usually migreations only update their own tree version).
-The ``ApdbSql`` schema version store in ``metadata`` will be ``0.1.0`` by default, but it can be changed via the command line option.
+It also populates metadata with version numbers for both ``schema`` and ``ApdbSql`` trees (usually migrations only update their own tree version).
+The ``ApdbSql`` schema version stored in ``metadata`` will be ``0.1.0`` by default, but it can be changed via the command line option.
 
 An example of specifying different version number for ApdbSql::
 
     $ apdb-migrate-sql upgrade --options apdb_sql_version=0.2.0 $APDB_URL schema_0.1.1
+
+
+Upgrade from 0.1.1 to 1.0.0
+===========================
+
+Migration script: `schema_1.0.0.py <https://github.com/lsst-dm/dax_apdb_migrate/blob/main/migrations/sql/schema/schema_1.0.0.py>`_
+
+This migration implements two unrelated schema changes:
+
+  - replacing ``ccdVisitId`` column in source tables with ``visit`` and ``detector`` columns (`DM-42435 <https://rubinobs.atlassian.net/browse/DM-42435>`_)
+  - replacing ``DiaSource.flags`` bitmask column with individual boolean columns for each flag (`DM-41530 <https://rubinobs.atlassian.net/browse/DM-41530>`_)
+
+Unpacking ``ccdVisitId`` value into ``visit`` and ``detector`` is implemented by specific instrument code and uses Butler registry to locate packer parameters.
+This requires passing additional parameters to the migration script specifying Butler repository and instrument name.
```

Comment:
Hmm, I'm not sure that any repo will do. The outputs do not matter, but I think that packer parameters are part of the instrument dimension records. Potentially different repos could have different parameters for the same instrument name (maybe unlikely but not impossible?) I think it would be safer to say that the same repo that was used for processing should be specified here.

Reply:
That's going to put a lot of stress on the caller (e.g. for precursor processing). However, it's true that the DECam definition [did change in 2021](https://github.com/lsst/obs_decam/commit/313493ebb600bb3b2fcab0cb93db9f9715214a04)...

Reply:
I added a line which says it's better to use original repo.